gateway/api/ray.py

```diff
@@ -186,6 +189,24 @@ def create_ray_cluster(
     raycluster_client = dyn_client.resources.get(
         api_version="v1alpha1", kind="RayCluster"
     )
+    if program_config:
```

Comment:
can we do this in jinja template? 

it will be something like
```python
manifest = cluster.render(
  {
    "cluster_name": cluster_name, 
    "user_id": user.username,
    "workers": config.workers,
    ...
  }
)
```

Reply:
I remember I tried jinja before and it didn't work.  I don't remember the reason.  I'll try if I can make it work here.

Reply:
I found that I found the solution of my issue before.  I change this to use ninja template.